loader-tests/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@slowreader/loader-tests",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "start": "tsx ./index.ts"
+  },
+  "dependencies": {
```

Comment:
Let’s move all dependencies to `devDependencies`.

During the production build we will run `pnpm install --prod` to reduce security risks from dependencies. So we will need as minimum as possible dependencies for production deploy.